### PR TITLE
[BREAKING CHANGES] Library Revamp

### PR DESCRIPTION
**Pull request by cjmaxik**
1. Simplified client init.
2. Added caching examples
3. Deleted old examples
4. Improved manual

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "truckersmp/api-client",
+  "name": "cjmaxik/api-client",
```
Comment (aidabeorn) [inline composer.json]: Can't commit this to this repository

Comment (cjmaxik) [reply composer.json]: Ouch...

Comment (cjmaxik) [reply composer.json]: Fixed

**Comment by cjmaxik** [comment]
Problems has been reviewed.

**Pull request merged**